src/lib/idp/introspectToken.ts

```diff
@@ -37,11 +43,22 @@ export async function introspectToken(token: string) {
           "Content-Type": "application/x-www-form-urlencoded",
         },
       },
-    )
-    .then((res) => res.data)
```

Comment:
👀 

Should we reopen the debate about promise chaining vs async await?
https://gcdigital.slack.com/archives/C01GFD150MC/p1711033765657599

Reply:
no no, try catch is good, sorry about that :)